core/utils.go

```diff
@@ -0,0 +1,29 @@
+package lucy
+
```

Comment:
Change name of the file from `utils` to something more descriptive

Reply:
utils.go contains utility functions used for different purposes. It is better to keep it as it is.